doc/developer/fpm.rst

```diff
@@ -0,0 +1,103 @@
+FPM
+===
+
+FPM stands for Forwarding Plane Manager and it's a ``zebra``'s daemon module.
```

Comment:
"it's a module for use with Zebra" or something

Reply:
Fixed!